themes/tiddlywiki/vanilla/base.tid

```diff
@@ -1055,6 +1055,19 @@ canvas.tc-edit-bitmapeditor  {
 	clear: both;
 }
 
+.tc-tiddler-body:before, .tc-tiddler-body:after {
+	content: "";
+	display: table;
+}
+ 
+.tc-tiddler-body:after {
+	clear: both;
+}
+ 
+.tc-tiddler-body {
+	zoom: 1;
```

Comment:
This is marked with IE6 / 7 ... which we can ignore, since TW won't work with them anyway.